src/doc/trpl/dining-philosophers.md

```diff
@@ -540,7 +540,7 @@ fn main() {
         Philosopher::new("Gilles Deleuze", 1, 2),
         Philosopher::new("Karl Marx", 2, 3),
         Philosopher::new("Emma Goldman", 3, 4),
-        Philosopher::new("Michel Foucault", 0, 4),
+        Philosopher::new("Michel Foucault", 4, 0),
```

Comment:
They haven't. It is in the explanation.
This is the examples way of trying to evade a deadlock.

Therefore I have reverted it :S